apm-sniffer/apm-sdk-plugin/spring-plugins/async-annotation-plugin/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <artifactId>spring-plugins</artifactId>
         <groupId>org.apache.skywalking</groupId>
-        <version>6.2.0-SNAPSHOT</version>
+        <version>6.3.0-SNAPSHOT</version>
```

Comment:
Why not use `${project.version}`?

Reply:
I think you can't set this in parent paragraph.

Reply:
> I think you can't set this in parent paragraph.

Oh yes, forgive me